sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizeBusinessCardOptions.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.Duration;
+
+/**
+ * Options that may be passed when using recognize receipt APIs on Form Recognizer client.
+ */
+@Fluent
+public final class RecognizeBusinessCardOptions {
+    private static final Duration DEFAULT_POLL_INTERVAL = Duration.ofSeconds(5);
```

Comment:
I think this needs to support locale or did you leave that out for me?

Reply:
I can add it.